lib/streamlit/watcher/local_sources_watcher.py

```diff
@@ -43,6 +43,8 @@ def __init__(self, session_data: SessionData):
         self._session_data = session_data
         self._on_file_changed: List[Callable[[], None]] = []
         self._is_closed = False
+        self._cached_sys_modules: Set[str] = set()
+        self._cached_modules_paths: Dict[str, Set[str]] = {}
```

Comment:
Is `_cached_modules_paths` actually used? (Looks like it's written to but not actually read.)

Reply:
Oops, was previously used but became unused after a late change